packages/cli/src/admin.js

```diff
@@ -27,7 +27,8 @@ async function decodeGovernorProposal(artifacts, id) {
 
 async function decodeAllActiveGovernorProposals(artifacts, web3) {
   const Voting = artifacts.require("Voting");
-  const voting = await Voting.deployed();
```

Comment:
This is basically the ripple change throughout the repo to force Truffle to use the overloaded methods (without the ancillary data prop). 

To remove this we need to make all downstream tests and infrastructure consider _only_ the ancillary methods and then we can desperate the overloaded method. This will enable us to roll back these test changes.